content/strategy-goals/strategy/code-graph/search/product.md

```diff
@@ -10,37 +10,71 @@ Quicklinks:
 
-We've launched [Notebooks](https://sourcegraph.com/notebooks?tab=explore) in public beta. You can create and share Notebooks with a free Sourcegraph cloud account. Read more about Notebooks in the [docs](https://docs.sourcegraph.com). This quarter will also see a [Code monitors](https://sourcegraph.com/code-monitoring?visible=4) release featuring a 10,000 repository limit on monitors (a 200x increase), major peformance improvements, multiple actions per monitor, Slack integration, and webhook support.
+In addition to the core search experience, Search Product team's scope includes [Code monitors](https://docs.sourcegraph.com/code_monitoring) and [Notebooks](https://sourcegraph.com/notebooks?tab=explore). Today, all products and features in Search Product's scope are included in every Sourcegraph contract.
 
 ## What's next and why
 
-#### Goals
+Over the next quarter, Search Product will focus on foundational work in critical product areas and conduct research to inform an improved search experience.
+
+### Foundational work
+
+Our planned foundational work falls broadly into two buckets: generalizing the search backend to make it more usable and powerful for internal consumers, and taking a fresh look at the core search UX.
+
+#### Search backend
+
+Over the past year, we’ve spent a significant amount of time cleaning up and improving the search backend. These efforts include, but are not limited to:
+
+- Implementing streaming across backends to minimize latency and memory usage
+- Partitioning search code into different domains and stages to manage complexity
+- Simplifying and unifying search result types
+
+The primary goal of these efforts has been to push the backend to a state where it will be maintainable in the face of new features. This is an ongoing process, but we’ve gained significant ground.
+
+Q2's work will start to shift the focus of backend work from maintainability to capability and flexibility. This means thinking of “search” more as an API that we provide for internal consumers that exposes all the information needed to build features on top of it. Though search is an API, it's not yet flexible enough to solve all the use cases presented by, for example, Code Insights, and some other proposed user-facing query features. This work is important because it enables development of high-value roadmap items, and the ability to conceive of novel features that weren't possible before.
+
+#### Search UX
+
+In light of new search capabilities, we believe there's a big opportunity to create an outstanding search experience, which is vital to Sourcegraph's continued dominance in the code search market.
+
+This foundational work will have design and technical components. Design will focus on digesting user feedback, conducting user research, and iterating on UI and UX solutions. Engineering will focus on enabling and collaborating with design to determine the execution path, specify a development roadmap, and start development.
```

Comment:
Isn't this also how any development team works? Or is there some unique aspect to what search core is doing here?

Reply:
I agree with Jason, I think we can do away with this paragraph (the effort is well-described in the research section).

```suggestion
```